science/model.py

```diff
@@ -338,9 +338,9 @@ class Interpreter(Dataclass):
         metadata=metadata(
             lambda: f"""An identifier to use in `#{{<id>...}}` placeholders.
 
-            The `#{{<id>}}` placeholder can be used a a placeholder [`command`](#{Ref(Command)})
-            fields to reference the interpreter distribution archive or `#{{<id>:<name>}}` to
-            reference named files provided by the interpreter distribution.
+            The `#{{<id>}}` placeholder can be used in [`command`](#{Ref(Command)}) fields to
```

Comment:
Nice. So in your fork you just go manually run the Deploy Doc Site job on your PR branch. That will fail 1st time because GH auto-configures a branch protection rule to only allow pages deploys from main. Kill the rule in your fork repo settings and then re-run. Voila - docs PRs previews hosted name-spaced and for "free": https://jsirois.github.io/lift/_/manifest/interpreter.html#id

N.B.: The Links icons are broken when not served from a domain root like science.a-scie.app. Here they are served under a `lift/` path. The same issue exists with the docsite served up from the local machine filesystem using `science doc open`. That is a ~vanity issue I'd like to fix.

Reply:
I'll add a note about this in CONTRIBUTING.md in a separate PR.

Reply:
Both fixed here: https://github.com/a-scie/lift/pull/37